src/data/detail-page/tabs/tabsInterfaces.ts

```diff
@@ -0,0 +1,11 @@
+export interface Category {
+    id: categoryId;
+    name: string;
+    text: string;
+}
+
+export type categoryId = 'learn' | 'jobs' | 'resources'
+
+export type categoriesModel = {
```

Comment:
Also here, let's rename the type to `CategoriesModel`

Reply:
Done :)